back/app/services/xlsx_service.rb

```diff
@@ -170,12 +177,17 @@ def generate_ideas_xlsx(ideas, view_private_attributes: false, with_tags: false)
   end
 
   def generate_initiatives_xlsx(initiatives, view_private_attributes: false)
+    custom_field_columns = CustomField.with_resource_type('User').order(:ordering)&.map do |field|
+      { header: multiloc_service.t(field.title_multiloc), f: ->(i) { i.author&.custom_field_values[field.key] } }
+    end
```

Comment:
See the comment about the user custom field below. Also, we are computing those even if they are not used. Wouldn't it be better to compute them in the `if view_private_attributes` to save a few requests to the DB?